assets/chat/css/style.scss

```diff
@@ -1565,3 +1565,25 @@ button.btn {
     background-color: $color-surface-light1;
     border-color: $color-surface-light3;
 }
+
+#importsettingsBtn {
+    display: none;
+}
+
+.importexport {
```

Comment:
Shouldn't these any other other variables with this name use kebab case ? (.import-export)
```
.importexport {
.importexport:hover {
```

Reply:
I suppose so, I'm still rusty with my variable naming